packages/create-sitecore-jss/src/templates/node-headless-ssr-experience-edge/src/config.types.ts

```diff
@@ -0,0 +1,53 @@
+import { LayoutServiceData } from '@sitecore-jss/sitecore-jss/layout';
```

Comment:
This file shouldn't be in the template, seems like added by mistake

Reply:
Do we keep it as it is or want me to transfer the types to config.ts?

Reply:
Probably, rename this file to -> `types.ts`, it has types not only related to the `config`